pkg/controller/installation/installer_test.go

```diff
@@ -16,13 +20,30 @@ import (
 	kubescheme "k8s.io/client-go/kubernetes/scheme"
 	"k8s.io/client-go/rest"
 	kubetesting "k8s.io/client-go/testing"
+	"k8s.io/helm/pkg/chartutil"
+	"k8s.io/helm/pkg/proto/hapi/chart"
 
 	shipper "github.com/bookingcom/shipper/pkg/apis/shipper/v1alpha1"
 	"github.com/bookingcom/shipper/pkg/controller"
 	"github.com/bookingcom/shipper/pkg/controller/janitor"
 	shippertesting "github.com/bookingcom/shipper/pkg/testing"
 )
 
+var localFetchChart = func(chartspec *shipper.Chart) (*chart.Chart, error) {
+	data, err := ioutil.ReadFile(
+		path.Join(
+			"testdata",
+			"chart-cache",
+			"localhost",
+			fmt.Sprintf("%s-%s.tgz", chartspec.Name, chartspec.Version),
```

Comment:
I'm of two minds with hardcoding localhost here. On one hand, it may never even matter. On the other hand, it doesn't match the actual RepoURL in the chartspec, which may make it for a very frustrating time for someone debugging around here down the road.